#! /usr/bin/env python

PACKAGE='jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("nmixtures", int_t, 0, "the number of gaussian mixture", 3, 1, 10)
gen.add("detect_shadows", bool_t, 0, "detect shadow", False)

exit(gen.generate(PACKAGE, "jsk_perception", "BackgroundSubstraction"))
